src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,193 @@
+        // Then
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        ExtractableResponse<Response> lineInfo = getAllLines();
+        List<String> nameList = lineInfo.body().jsonPath().getList("name", String.class);
+        assertThat(nameList).contains(newLineName);
+    }
+
+    /**
+     * Given: 여러 개의 지하철 노선이 등록되어 있고,
+     * When: 관리자가 지하철 노선 목록을 조회하면,
+     * Then: 모든 지하철 노선 목록이 반환된다.
+     */
+    @DisplayName("지하철 노선 목록을 조회한다.")
+    @Test
+    void testGetAllLine() {
+        // Given
+        String lineName1 = "사당";
+        String lineName2 = "방배";
+        createLine(new LineRequest(lineName1, "red", stationId1, stationId2, 10L));
+        createLine(new LineRequest(lineName2, "blue", stationId3, stationId4, 10L));
+        // When
+        ExtractableResponse<Response> lineInfo = getAllLines();
+        // Then
+        List<String> lineNames = lineInfo.body().jsonPath().getList("name", String.class);
+        assertThat(lineNames).containsExactly(lineName1, lineName2);
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 조회하면,
+     * Then: 해당 노선의 정보가 반환된다.
+     */
+    @DisplayName("지하철 노선을 조회한다.")
+    @Test
+    void testGetLine() {
+        // Given
+        String lineName1 = "사당";
+        ExtractableResponse<Response> response = createLine(new LineRequest(lineName1, "red", stationId1, stationId2, 10L));
+        // When
+        String url = response.response().getHeader("Location");
+        ExtractableResponse<Response> lineInfo = getLine(url);
+        // Then
+        String lineName = lineInfo.body().jsonPath().get("name");
+        assertThat(lineName).isEqualTo(lineName1);
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 수정하면,
+     * Then: 해당 노선의 정보가 수정된다.
+     */
+    @DisplayName("지하철 노선을 수정한다.")
+    @Test
+    void testUpdateLine() {
+        // Given
+        String previousLineName = "사당";
+        String newLineName = "방배";
+        ExtractableResponse<Response> createLineResponse = createLine(new LineRequest(previousLineName, "red", stationId1, stationId2, 10L));
+        // When
+        Map<String, String> updateLineInfoParam = Map.of("name", newLineName, "color", "red");
+        updateLine(updateLineInfoParam, 1L);
+        // Then
+        ExtractableResponse<Response> lineInfo = getLine(createLineResponse.response().getHeader("Location"));
+        String lineName = lineInfo.body().jsonPath().get("name");
+        assertThat(lineName).isEqualTo(newLineName);
+    }
+
+    /**
+     * Given: 특정 지하철 노선이 등록되어 있고,
+     * When: 관리자가 해당 노선을 삭제하면,
+     * Then: 해당 노선이 삭제되고 노선 목록에서 제외된다.
+     */
+    @DisplayName("지하철 노선을 삭제한다.")
+    @Test
+    void testRemoveLine() {
+        // Given
+        String lineName1 = "사당";
+        ExtractableResponse<Response> createLineResponse = createLine(new LineRequest(lineName1, "red", stationId1, stationId2, 10L));
```

Comment:
```suggestion
        var 사당 = createLine(new LineRequest("사당", "red", stationId1, stationId2, 10L));
```

사당으로 정확히 표현해 주는 게 어떨까요? 인수 테스트가 봐야 하는 대상이 누구인지 한번 지난 영상을 통하여 확인하여 대상자가 되어 인수 테스트를 확인하는 것도 중요하다고 봐요.